study-html/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Progate</title>
+    <link rel="stylesheet" href="stylesheet.css">
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <div class="header-left">
+          <img class="logo" src="https://prog-8.com/images/html/advanced/main_logo.png">
+        </div>
+        <div class="header-right">
+          <a href="#" class="login">ログイン</a>
+        </div>
+      </div>
+    </header>
+    <div class="top-wrapper">
```

Comment:
`two-wrapper` はあまりにもわかりにくいです。ここは `header` などと同じ階層になるので、同じセマンティックなタグがある場合はそれを使いましょう。ない場合には、class名に `*-section` といったように一つのセクションであることを示しましょう。
```suggestion
    <div class="hero-section">
```